.vscode/settings.json

```diff
@@ -5,5 +5,10 @@
     },
     "search.exclude": {
         "out": true // set this to false to include "out" folder in search results
-    }
-}
\ No newline at end of file
+    },
+    "[typescript]": {
+      "editor.tabSize": 4,
+      "editor.insertSpaces": true,
+      "editor.detectIndentation": false
+  }
```

Comment:
This brace is misaligned - transgression!! :smiling_imp:

Reply:
response is classified ;)